### DISCUSSION TITLE
Firmware v1.6.2

### DISCUSSION THREAD

**Discussion by tjko** [tjko/fanpico#82] [Announcements]
## What's Changed
* Fix possible hang during boot (introduced in v1.6.1 firmware) if flash filesystem needed to be initialized. 
* New command: 
  * SYS:LFS:FORMAT

**Full Changelog**: https://github.com/tjko/fanpico/compare/v1.6.1...v1.6.2

<hr /><em>This discussion was created from the release <a href='https://github.com/tjko/fanpico/releases/tag/v1.6.2'>Firmware v1.6.2</a>.</em>

**Comment by ianziot** [comment]
Hi tjko, I just spent an hour looking over your github site for your fan controller and would like to commend your effort in this project. I am a 71yr old man from New Zealand and began teaching myself about digital electronics about 3 yrs ago. I have a largely finished a CNC camera mount project that I wish to document using a github site and was looking for an example of similar mixed hardware ans software sites when I stumbled across your project. In fact I have no use for such a sophisticated fan controller but admire your work anyway. I see that you have used as a RPi as the primary board whereas I have used an ESP32 Devkit board. Of particular interest is the way that you have documented the PCB's using images and pdf files as well as the kicad source. Just saying hi, and well done.

**Reply by tjko** [reply]
Glad you found this useful. I've build over the years several custom fan controllers, but never bothered to document them. This time I thought to spend little extra time to try to document everything.  I went with RP2040 microcontroller instead of ESP32 as Pico was just released when I started on this. So this became also a  project to learn about features of the RPi Pico (and Pico W).
